apps/ledger-live-mobile/e2e/specs/addAccounts/bitcoin.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { expect, device } from "detox";
+import { DeviceModelId } from "@ledgerhq/devices";
+import { loadBleState, loadConfig } from "../../bridge/server";
+import PortfolioPage from "../../models/wallet/portfolioPage";
+import DeviceAction from "../../models/DeviceAction";
+import AccountsPage from "../../models/accounts/accountsPage";
+import AddAccountDrawer from "../../models/accounts/addAccountDrawer";
+import { getElementByText } from "../../helpers";
+
+let portfolioPage: PortfolioPage;
+let deviceAction: DeviceAction;
+let accountsPage: AccountsPage;
+let addAccountDrawer: AddAccountDrawer;
+
+const knownDevice = {
+  name: "Nano X de test",
+  id: "mock_1",
+  modelId: DeviceModelId.nanoX,
+};
+
+describe("Add Bitcoin Accounts", () => {
+  beforeAll(async () => {
```

Comment:
I would add an `afterAll` reenabling the detox synchronisation just to make sure it doesn't affect the other tests